getting_started/step_by_step/intro_to_the_editor_interface.rst

```diff
@@ -11,92 +11,103 @@ Project manager
 ---------------
 
 When you launch Godot, the first window you’ll see is the Project
-Manager. It lets you create, remove, import or play game projects.
+Manager. Since you have no projects there will be a popup asking if you
+want to open the asset library, just click cancel, we'll look at it later.
 
-|image0|
+.. image:: img/project_manager_first_open.png
+
+Now you should see the project manager. It lets you create, remove, import
+or play game projects.
+
+.. image:: img/editor_ui_intro_project_manager_01.png
 
 In the top-right corner you’ll find a drop-down menu to change the
 editor’s language.
 
-|image1|
+.. image:: img/editor_ui_intro_project_manager_02.png
 
 From the **Templates** tab you can download open source project templates and
-demos to help you get started faster. Just select the template or demo you want,
-click download, once it's finished downloading click install and choose where
-you want the project to go.
+demos from the Asset Library to help you get started faster. Just select the
+template or demo you want, click download, once it's finished downloading click
+install and choose where you want the project to go. You can learn more about
+the asset library here :ref:`doc_what_is_assetlib`
```

Comment:
Missing dot at the end of the line.

Reply:
And generally this won't look good, it will be: "You can learn more about the asset library here About the Asset Library".

This should indeed be something like:
```
You can learn more about it in :ref:`doc_what_is_assetlib`.
```